src/components/Users/Users.js

```diff
@@ -28,35 +27,10 @@ let Users = (props) => {
           <div>
             {u.followed
               ? <button disabled={props.followingInProgress.some(id => id === u.id)} onClick={() => {
-                props.toggleFollowingProgress(true, u.id);
-                axios.delete(`https://social-network.samuraijs.com/api/1.0/follow/${u.id}`, {
-                  withCredentials: true,
-                  headers: {
-                    "API-KEY": "554413f3-0efe-4111-880a-a4852cf6f5a8"
-                  }
-                })
-                  .then(response => {
-                    if (response.data.resultCode === 0) {
-                      props.unfollow(u.id);
-                    };
-                    props.toggleFollowingProgress(false, u.id);
-                  });
-
+                props.unfollow(u.id);
```

Comment:
Looks pretty nice, you move complex logic in a function and decrease system difficult, now you code is easier for reading and understanding.

Reply:
Do you know hooks? `useCallback` and `useMemo`? You also should wrap `() => { props.unfollow(u.id) }` in useCallback for learning purposes.